packages/launch/src/hooks/use-cart.ts

```diff
@@ -26,7 +33,6 @@ export function useCart(): { goToCheckout: () => Promise< void > } {
 	const goToCheckout = async () => {
 		// setting the cart with Launch products can be extracted
 		// to an action creator on the Launch data-store
-		const planProduct = plan && getPlanProduct( plan, flow );
```

Comment:
The `planProduct` had also a cart specific property `extra: { source: flow }`
https://github.com/Automattic/wp-calypso/pull/48790/files#diff-5cfb12d41e1a3c60d3cf55652ecb70b00e08e5fa8e726cfc4cd79ef32e0b5170L31-L33

So we can rename the utility function but probably it's good to keep using it since `flow` has nothing to do with `PLANS_STORE`.